assets/scss/components/_tree.scss

```diff
@@ -43,14 +43,25 @@
         background-color: $color-ui-hover;
     }
 
-    span {
+    &:focus-within {
```

Comment:
this is not supported in some older browsers that we should support with our browserslist config (which we don't actually have, since we use the default config)

Reply:
Good point, no longer relying on this